perf/benchmark/run_benchmark_job.sh

```diff
@@ -117,12 +117,12 @@ function setup_fortio_and_prometheus() {
 
 #TODO: add stackdriver filter
 function prerun_v2_nullvm() {
-  export SET_OVERLAY="values.telemetry.enabled=true,values.telemetry.v1.enabled=false,values.telemetry.v2.enabled=true,values.telemetry.v2.prometheus.enabled=true"
+  export SET_OVERLAY="--set values.telemetry.enabled=true --set values.telemetry.v1.enabled=false --set values.telemetry.v2.enabled=true --set values.telemetry.v2.prometheus.enabled=true"
```

Comment:
this is now the default, so hopefully on the other tests we explicitly turn on mixer and turn off v2?

Reply:
yeah, if telemetryv2 is turn on by default, then when we test mixer config, we did nothing, we need to explicitly turn on mixer to test mixer configs.

Reply:
yep I just added explicit overwrite for them